### PR TITLE
Introduce issue templates

### PR DESCRIPTION
**Pull request by raphaelrobert**
This PR introduces issue templates. This is working towards #196.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Use the following API call/function: ...
+2. See error
+
+**Expected behavior**
```
Comment (kkohbrok) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: In addition to "Expected Behaviour", maybe have a field called "Actual Behaviour"?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
+A brief explanation of what happens instead.
+
+**Screenshots or debug log**
+If applicable, add screenshots or debug logs to help explain your problem.
+
+**Platform (please complete the following information):**
+ - OS: [e.g. macOS]
```
Comment (franziskuskiefer) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: OS with version would be even better.

**Review by franziskuskiefer — COMMENTED** [review]
Thanks! A couple thought, no strong opinions.

Maybe give all of them a header with something like 
"Issue tracker is **ONLY** used for reporting bugs, feature requests, and spec changes. For support or discussions please use https://github.com/openmls/openmls/discussions (or some chat service)"
Also, I'm not sure we want feature requests in here. They are usually better handled in discussions before filing an issue.

There are different approaches to how to put the text in. I like when they are commented out (`<!--- -->`) because they can stay in when writing the ticket.

**Comment by raphaelrobert** [comment]
I addressed your remarks.
I left feature requests in, because not everything fits into the bug/spec change categories. We can try this for ourselves and change this later if we see it doesn't work.

**Review by franziskuskiefer — APPROVED** [review]

**Pull request merged**